lms/static/sass/shared/_course_object.scss

```diff
@@ -7,33 +7,44 @@
       width: flex-grid(4);
       @include margin-right(flex-gutter());
       float: left;
-
-      &:nth-child(3n+3) {
-        @include margin-right(0);
-      }
     }
 
     .courses-listing {
+      @include outer-container;
       @include clearfix();
       margin: 0;
       padding: 0;
       list-style: none;
 
       .courses-listing-item {
-        width: flex-grid(4);
-        @include margin-right(flex-gutter());
-        @include float(left);
+        height: 360px;
+
+        @include media($mobile) {
+          @include span-columns(4);
+          margin: 15px 0px;
+        }
+
+        @include media($tablet) {
+          @include span-columns(4);
+          margin: 15px 3px 15px;
+        }
 
-        &:nth-child(3n+3) {
-          @include margin-right(0);
+        @include media($desktop) {
+          @include span-columns(4);
+          margin: 15px 5px 15px;
         }
+
+        @include media($xl-desktop) {
+          @include span-columns(3);
+          margin: 15px 7px 15px;
+        }
+
       }
     }
 
     .course {
       background: $body-bg;
-      border: 1px solid $border-color-1;
-      border-radius: 2px;
+      border-bottom: 3px solid #3185bd;
       @include box-sizing(border-box);
       box-shadow: 0 1px 10px 0 rgba(0,0,0, 0.15), inset 0 0 0 1px rgba(255,255,255, 0.9);
```

Comment:
Not your doing, but please convert these color values to the known transparent Sass variables - https://github.com/edx/edx-platform/blob/mjevtic/SOL-531/lms%2Fstatic%2Fsass%2Fbase%2F_variables.scss#L61 and https://github.com/edx/edx-platform/blob/mjevtic/SOL-531/lms%2Fstatic%2Fsass%2Fbase%2F_variables.scss#L70).